verticapy/connection/write.py

```diff
@@ -145,8 +204,23 @@ def new_connection(
         associated environment variables. VerticaPy reads
         the associated environment variables instead of
         writing and directly using the username and password.
-        For example: {'user': 'ENV_USER',
-                      'password': 'ENV_PASSWORD'}
+
+        For example: {'user': 'ENV_USER', 'password'
+        : 'ENV_PASSWORD'}
+
+    Example
+    -------
+    .. code-block:: python
+
+        from verticapy.connection import new_connection
+
+        conn_info = {"host": "10.211.55.14",
+                     "port": "5433",
+                     "database": "testdb",
+                     "password": "XxX",
+                     "user": "dbadmin"}
+
+        new_connection(conn_info, name = "VerticaDSN")
```

Comment:
add one line to explain what you do. If there is no output you can write a note like:

.. note:: If no errors are raised, the new connection was successful.